rviz_common/src/rviz_common/config.cpp

```diff
@@ -282,14 +282,14 @@ bool Config::mapGetString(const QString & key, QString * value_out) const
 
 void Config::makeValid()
 {
-  if (node_.get() == NULL) {
+  if (node_.get() == nullptr) {
```

Comment:
@clalancette Any reason not to change it like this?

```suggestion
  if (!node_) {
```

Reply:
Mostly this was a search-n-replace, so I didn't consider it.

I personally prefer the more explicit code for readability.  It's really easy for me to miss the `!` when glancing over code, while the `== nullptr` stands out to me.  But if others feel strongly about it, I can change it.

Reply:
I don't mind the `nullptr`, but the `.get()` feels kinda silly.

Up to you though, it's really not important in any sort of capacity.

Reply:
+1 to remove the `.get()` (which doesn't seem necessary).

Reply:
All right, I've gone ahead and removed the `.get()`.  I'll run CI now.